timeline.py

```diff
@@ -0,0 +1,35 @@
+import fileinput
+
+
+class Timeline:
+
+    # TODO: Once the program is moved to .git, change the path to undo.timeline
+    def __init__(self, path='.git/undo.timeline'):
+        self._path = path
+
+    # Add the line to end of file
+    def add(self, command_list):
+        new_line = ' '.join(command_list) + '\n'
+        with open(self._path, 'a') as file:  # a = append mode
+            file.write(new_line)
+
+    # Return the last line of the file
+    # Return None if file is not found or empty
+    def get_last_line(self):
+        line = None
+        try:
+            with open(self._path, 'r') as file:
+                for line in file:  # get last line
+                    pass
+        except IOError:  # no such file
+            return None
```

Comment:
ok, tried to add the exception, but it makes the code more messy. i also prefer not to raise exception because this is not really and error; rather, this can happen for the first time user type `git undo` without any previous `git commit`, so it's not really error-ing